src/containers/NoteArea/NoteCard/NoteCard.js

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 import { NoteItems } from "../../";
 class NoteCard extends React.Component {
-  constructor({ note }) {
-    super(note);
+  constructor(props) {
+    super(props);
```

Comment:
React threw an error when I tried destructering here. Maybe there I am overlooking something. Would be nice to destructure here and not get an error message :(

Reply:
Not sure why destructuring didn't work for you there. We can look at it in our refactor before the weekend.